test-providers.d/io-github-autotest-libvirt.ini

```diff
@@ -1,5 +1,5 @@
 [provider]
-uri: https://github.com/autotest/tp-libvirt.git
```

Comment:
@kylazhang looks like local commit, can you remove it thanks.

Reply:
Fixed, thanks